test/github-issues/4440/issue-4440.ts

```diff
@@ -0,0 +1,57 @@
+import "reflect-metadata";
+import { Connection } from "../../../src/connection/Connection";
+import { closeTestingConnections, createTestingConnections, reloadTestingDatabases } from "../../utils/test-utils";
+import { Post } from "./entity/Post";
+
+describe("github issues > #4440 simple-json column type throws error for string with no value", () => {
+
+    let connections: Connection[];
+    before(async () => {
+        connections = await createTestingConnections({
+            entities: [Post],
+            schemaCreate: true,
+            dropSchema: true,
+            enabledDrivers: [
```

Comment:
Removing `enabledDrivers` will enable all drivers. We do not want to update this array every time new driver is added.

Reply:
done